controllers/replicationdestination_controller.go

```diff
@@ -77,7 +81,6 @@ type ReplicationDestinationReconciler struct {
 //+kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=rolebindings,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,resourceNames=scribe-mover,verbs=use
```

Comment:
Since the static manifests `config/openshift/mover_scc.yaml` will retain the fixed `scribe-mover` name for the SCC, this should probably be left here since we need the static yamls to all agree when someone uses `make deploy-openshift`
(and these kubebuilder markers are used to automatically generate the RBAC)

Reply:
ah, i added them back
the only reason i removed them was _something_ was failing w/ them there - `make something` i cannot remember what